packages/views.py

```diff
@@ -3,13 +3,16 @@
 from rest_framework.response import Response
 
 from packages.modules import npm
-from packages.serializers import VersionedPackageSerializer
+from packages.serializers import PackageSerializer
 
 
 class PackageView(views.APIView):
     renderer_classes = [renderers.JSONRenderer]
 
-    def get(self, request: Request, package_name: str, version: str):
-        package_info = npm.get_versioned_package(package_name, version)
-        serializer = VersionedPackageSerializer(package_info)
+    def get(self, request: Request, package_name: str, range: str | None = None):
+        if range is None:
+            range = "*"
+
+        package_info = npm.get_package(package_name, range)
```

Comment:
```suggestion
    def get(self, request: Request, package_name: str, range: str = "*"):
        package_info = npm.get_package(package_name, range)
```

Unnecessary conversion of types/branching logic here when we could explicitly set the default value.